CMakeLists.txt

```diff
@@ -249,7 +258,7 @@ else()
 endif()
 
 # Set architecture alignment requirements
-if(BASEARCH_ARM_FOUND OR BASEARCH_X86_FOUND)
+if(BASEARCH_ARM_FOUND OR BASEARCH_X86_FOUND OR (BASEARCH_PPC_FOUND AND "${ARCH}" MATCHES "powerpc64le"))
```

Comment:
@nmoinvaz should I put this in alphabetical order too, or leave it in the end, since it has an extra check?

Reply:
I wasn't as concerned about that one, but I'd say alphabetical too.

Reply:
Ok. Changed.